content/methods/discover/stakeholder-and-user-interviews.md

```diff
@@ -29,7 +29,7 @@ method:
 
 ## Additional resources
 
-- [Interview checklist]({{ "/methods/interview-checklist/" | url }})
+- [Interview checklist]({{ "https://guides.18f.gov/ux-guide/interview-checklist/" | url }})
```

Comment:
We do not want to put in the base url (in this case, `https://guides.18f.gov`) as base urls are automatically inserted by 11ty depending on the way 11ty is set up. Often we work on these files locally and we want the base url to point to our own local instance, rather than redirect to guides.18f.gov proper. 

In other words, You can replace `methods` with `ux-guide` safely here.

Reply:
Got it. I did have that question. Thanks for clarifying!